public/javascripts/registration-validation.js

```diff
@@ -0,0 +1,61 @@
+var errorNames = [];
+var validator = new FormValidator('registration_form', [{
+  name: 'firstname',
+  display: 'required',
+  rules: 'required|alpha'
+},
+  {
+    name: 'lastname',
+    display: 'required',
+    rules: 'required|alpha'
```

Comment:
remove the validation on firstname and lastname.

I could not sign up to yetu with these constraints. It does not accept `é`.

Is there a requirement to force users to only use ASCII characters for their name? Also, some names include `-` for example. Get rid of this, it's unhelpful, unnecessary, and excludes all the people who do not have an ASCII name, which is probably >80% of the world population.